src/pip/_internal/commands/freeze.py

```diff
@@ -75,6 +70,7 @@ def __init__(self, *args, **kw):
             action='store_true',
             help='Exclude editable package from output.')
 
+        self.cmd_opts.add_option(cmdoptions.list_path())
```

Comment:
Does where `add_option()` is called control the order of display when invoking `--help`? If so, would it make sense to group `--path` after `--local` and `--user` like it was before (since `--path` is mutually exclusive with those other options)?

Reply:
You are totally right. Fixed.